macros/utils/table_operations/insert_rows.sql

```diff
@@ -111,12 +111,22 @@
     {{- return(string_value | replace("'", "''")) -}}
 {%- endmacro -%}
 
+{%- macro athena__escape_special_chars(string_value) -%}
+    {{- return(string_value | replace("'", "''")) -}}
+{%- endmacro -%}
+
 {%- macro render_value(value) -%}
     {%- if value is defined and value is not none -%}
         {%- if value is number -%}
             {{- value -}}
         {%- elif value is string -%}
-            '{{- elementary.escape_special_chars(value) -}}'
```

Comment:
inserts with timestamp strings should be casted explicitly in Athena. How to handle it properly?

Reply:
thanks @haritamar , looks really straightforward. Do you see any problem with using `edr_cast_as_timestamp` for timestamp literals? It's needed for Athena to add `timestamp` type but will also change timestamp literals for other connector. Or should I add a dedicated macro to handle timestamp literals which would be no-op for all but Athena?